aws-lambda/dotnet/src/function.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Would this work for .NET 6? I think it might??

Reply:
Yea I was looking at that, though due to a serious level of ignorance regarding Lambda, I didn't really understand what I was reading 😆. So, I'd imagine this component might compile just fine against .NET 6, but then it'd fail to deploy or something - does that sound about right?

Reply:
Yeah, with .NET and Go the managed layer is just the OpenTelemetry Lambda collector. Other runtimes include the SDK and other things. So this should just mostly work in .NET 6 with only minor changes. But the managed layer needs to be updated to add .NET 6 as a supported runtime.